app/models/ability.rb

```diff
@@ -0,0 +1,34 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return unless user.present?
+
+    can :manage, Expenditure, author: user
+    can :manage, Category, user:
+    # Define abilities for the user here. For example:
+    #
+    #   return unless user.present?
+    #   can :read, :all
+    #   return unless user.admin?
+    #   can :manage, :all
+    #
+    # The first argument to `can` is the action you are giving the user
+    # permission to do.
+    # If you pass :manage it will apply to every action. Other common actions
+    # here are :read, :create, :update and :destroy.
+    #
+    # The second argument is the resource the user can perform the action on.
+    # If you pass :all it will apply to every resource. Otherwise pass a Ruby
+    # class of the resource.
+    #
+    # The third argument is an optional hash of conditions to further filter the
+    # objects.
+    # For example, here the user can only update published articles.
+    #
+    #   can :update, Article, published: true
+    #
+    # See the wiki for details:
+    # https://github.com/CanCanCommunity/cancancan/blob/develop/docs/define_check_abilities.md
+  end
```

Comment:
- [ ] I'm getting this syntax error when I try to create a new transaction. Please fix that. 🙏 

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/221332962-27478309-496b-4b78-a57e-d84d94d99c03.png)

Reply:
- [ ] As requested by the previous reviwer kindly fix this issue as I am also unable to create a new transaction
![image](https://user-images.githubusercontent.com/81932478/221373399-89b00441-ff6f-4429-8c90-53ed232ab6ae.png)